strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/main/java/org/carlspring/strongbox/nuget/file/Dependencies.java

```diff
@@ -0,0 +1,80 @@
+package org.carlspring.strongbox.nuget.file;
+
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import javax.xml.bind.annotation.XmlElement;
+
+public class Dependencies
+        implements Serializable
+{
+
+    @XmlElement(
```

Comment:
Cosmetic: Same as my earlier comment regarding the annotations. (Would you mind applying the change across your pull request? Thanks! :) ).